### PR TITLE
dodalem bootstrapa

### PR DESCRIPTION
**Pull request by valentinebaliuk**
Hej, taki próbny pull z dodanym bootstrapem

### CODE REVIEW

File: izp/polls/templates/polls/detail.html [hunk 1]
```diff
@@ -1,16 +1,26 @@
-<h1>{{ question.question_text }}</h1>
+<!DOCTYPE html>
+<html lang="pl"> 
+	<head>
```
Comment (czapiga) [inline izp/polls/templates/polls/detail.html]: Inconsistent formatting. Please use spaces instead of tabs

File: izp/polls/templates/polls/index.html [hunk 2]
```diff
@@ -1,15 +1,51 @@
...
-        <a href="{% url 'polls:detail' question.id %}">{{ question.question_text }}</a>
-        {{question.start_date}} - {{question.end_date}}
-        <a href="{% url 'polls:result' question.id %}">Wynik</a>
-    </li>
-    {% endfor %}
-</ul>
-{% else %}
-<p>Brak ankiet.</p>
-{% endif %}
+<!DOCTYPE html>
```
Comment (czapiga) [inline izp/polls/templates/polls/index.html]: You've indented code way too much. It's hard to read with all these whitespaces

File: izp/polls/templates/polls/detail.html (hunk 1 above)
Comment (valentinebaliuk) [reply izp/polls/templates/polls/detail.html]: oh, ok.
and i think that we need to create text file with our project code conventions .

Comment (czapiga) [reply izp/polls/templates/polls/detail.html]: I will try to add code style check in CI that I will enable in this week.

File: izp/izp/settings.py [hunk 3]
```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['jec18d2y.apps.lair.io']
```
Comment (jasiekmarc) [inline izp/izp/settings.py]: This seems to be an artifact coming from the fancy on-line editor, you are using. It will make Django refuse requests if it listens on any domain other than `jec18d2y.apps.lair.io`.

**Comment by czapiga** [comment]
Actually I don't know what to do with this task.. I'm unable to check all of that imported code. I don't know what is really needed.. I think that maybe it was overkill to use bootstrap for our site. Every day I'm more convinced that we can create simple .css that will work just fine for our needs. Please leave this pull request for now and start working on simple single .css file. I will also inform you about this task in next few days using mailing list.

**Comment by baliukvalentine** [comment]
Ok, we can use clear css

W dniu sobota, 11 listopada 2017 czapiga <notifications@github.com>
napisał(a):

**Comment by baliukvalentine** [comment]
Well, now i have little bit more time to write.
I can write custom css lib for our project, but for what? We have a lot of
ready to use libraries.
In the same way we can said that it's bad idea to use Django because there
is a lot of unchecked code.
Bootstrap it is tested by timelibrary, also used in "System zapisów ii.uwr"
project..
But ok, if you wanna use something more light we can use Pure.css.



W dniu sobota, 11 listopada 2017 Valentine Baliuk <
valentine.baliuk@gmail.com> napisał(a):

**Comment by czapiga** [comment]
Please fix conflicts - merge with master on your branch and update PR

**Comment by czapiga** [comment]
I've just checked your PR and it's not working. Please check if all fields in form have proper names. Have you tested it before updating review?

File: izp/izp/settings.py [hunk 4]
```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['kgzm4y7u.apps.lair.io']
```
Comment (czapiga) [inline izp/izp/settings.py]: revert this change

File: izp/polls/static/css/custom.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    text-decoration: none;
+    color: white !important;
+}
+
+.left-nav a:hover {
+    background-color: #111;
+}
+.right-nav:hover {
+    background-color: #111;
+}
```
Comment (czapiga) [inline izp/polls/static/css/custom.css]: Add new line at the end of file

File: izp/polls/templates/admin/base.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        <br class="clear" />
+    </div>
+    <!-- END Content -->
+
+    {% block footer %}<div id="footer"></div>{% endblock %}
+</div>
+<!-- END Container -->
+
+</body>
+</html>
```
Comment (czapiga) [inline izp/polls/templates/admin/base.html]: Add new line at the end of file

File: izp/polls/templates/admin/base_site.html [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{% extends "admin/base.html" %}
+
+{% block title %}{{ title }} | System głosowania{% endblock %}
+{% block extrastyle %} {% endblock %}
+{% block branding %}
+<h1 id="site-name"><a href="{% url 'admin:index' %}">System głosowania</a></h1>
+{% endblock %}
+
+{% block nav-global %}{% endblock %}
```
Comment (czapiga) [inline izp/polls/templates/admin/base_site.html]: Add new line at the end of file

File: izp/polls/templates/polls/base.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+      <link href={% static "css/bootstrap.css" %} rel="stylesheet">
+      <link href={% static "css/custom.css" %} rel="stylesheet">
+   </head> 
+   <body>
+       <div class="header">
+           <img src={% static "image/logo.jpg" %} />
+       </div>
+       <div class="navbar">
+          <div class="left-nav"><a href="{% url 'polls:index' %}">Głosowanie</a></div>
+          <div class="left-nav"><a href="#stat">Statystyki</a></div>
```
Comment (czapiga) [inline izp/polls/templates/polls/base.html]: Please remove not needed links

File: izp/polls/templates/polls/index.html [hunk 9]
```diff
@@ -1,15 +1,35 @@
...
+         {% endfor %}
+      </tbody>
+   </table>
+   {% else %}
+   <div class="text-info" role="alert">
+      <h2>Brak ankiet!</h2> 
+      <h4>Nie ma żadnego otwartego głosowania na ten moment.</h4>
+   </div>
+   {% endif %}
+{% endblock %}
```
Comment (czapiga) [inline izp/polls/templates/polls/index.html]: Add new line at the end of file

File: izp/polls/templates/polls/index.html [hunk 10]
```diff
@@ -1,15 +1,35 @@
...
-<p>Brak ankiet.</p>
-{% endif %}
+{% extends 'polls/base.html' %}
+{% block content %} 
+   {% if questions_list %}
+   <table class="table table-striped">
+      <caption><h3>Lista pytań</h3></caption>
+      <thead>
+         <tr>
+            <th>Glosowanie</th>
```
Comment (czapiga) [inline izp/polls/templates/polls/index.html]: Glosowanie -> Głosowanie

File: izp/polls/templates/polls/result.html [hunk 11]
```diff
@@ -1,23 +1,45 @@
...
+{% extends 'polls/base.html' %}
+{% block content %}
+   
+   <h1>{{ question.question_text }}</h1>
+  
+   <table class="table table-bordered">
+      <caption><h3>Wynik głosowania:</h3></caption>
+      <thead>
+         <tr>
+            <th>Wybór</th>
```
Comment (czapiga) [inline izp/polls/templates/polls/result.html]: Wybór -> Odpowiedź

File: izp/polls/templates/polls/result.html [hunk 12]
```diff
@@ -1,23 +1,45 @@
...
+         <tr>
+            <td>{{ code.code }}</td>
+            <td>{{ code.num_of_votes }}</td> 
+            <td>{{ code.last_choice }}</td>
+         </tr>
+         {% endfor %}
+      </tbody>
+   </table>
+   {% endif %}
+{% endblock %}
```
Comment (czapiga) [inline izp/polls/templates/polls/result.html]: Add new line at the end of file

File: izp/polls/templates/polls/index.html [hunk 13]
```diff
@@ -1,15 +1,35 @@
...
+               <a href="{% url 'polls:result' question.id %}">Wynik{{question.wynik}}</a>
+            </td> 
+         </tr>
+         {% endfor %}
+      </tbody>
+   </table>
+   {% else %}
+   <div class="text-info" role="alert">
+      <h2>Brak ankiet!</h2> 
+      <h4>Nie ma żadnego otwartego głosowania na ten moment.</h4>
```
Comment (czapiga) [inline izp/polls/templates/polls/index.html]: I think lines 31,32 are redundant. One would be enough.

File: izp/polls/templates/polls/index.html [hunk 14]
```diff
@@ -1,15 +1,35 @@
...
+         {% for question in questions_list %}
+         <tr>
+            <td>
+               <a href="{% url 'polls:detail' question.id %}">
+                  {{question.question_text}}
+               </a>
+            </td>
+            <td>{{question.start_date}} - {{question.end_date}}</td>	
+            <td>
+               <a href="{% url 'polls:result' question.id %}">Wynik{{question.wynik}}</a>
```
Comment (czapiga) [inline izp/polls/templates/polls/index.html]: ```{{question.wynik}}``` ? Please remove

File: izp/polls/templates/admin/base.html [hunk 15]
```diff
@@ -0,0 +1,94 @@
+{% load i18n static %}<!DOCTYPE html>
```
Comment (czapiga) [inline izp/polls/templates/admin/base.html]: It looks like you've just copied some code. Please revert changes from izp/polls/templates/admin/base.html and izp/polls/templates/admin/base_site.html.

File: izp/polls/templates/polls/result.html [hunk 16]
```diff
@@ -1,23 +1,45 @@
...
-    {% for choice in choices %}
-    <li>{{ choice.choice_text }} : {{ choice.votes }}</li>
-    {% endfor %}
-</ul>
+{% extends 'polls/base.html' %}
+{% block content %}
+   
+   <h1>{{ question.question_text }}</h1>
+  
+   <table class="table table-bordered">
```
Comment (czapiga) [inline izp/polls/templates/polls/result.html]: "Wynik głosowania" and "Użyte kody" tables look different. Please use the same table type in both cases (I prefer ```<table class="table table-striped">```)
